src/common/xoa-upgrade.js

```diff
@@ -0,0 +1,47 @@
+import Icon from 'icon'
+import Link from 'react-router/lib/Link'
+import React from 'react'
+import { Card, CardHeader, CardBlock } from 'card'
+import { getXoaPlan, propTypes } from 'utils'
+
+export const Upgrade = propTypes({
+  available: propTypes.number,
+  place: propTypes.string
+})(({
+  available,
+  place
+}) => (
+  <Card>
+    <CardHeader>Upgrade needed</CardHeader>
+    <CardBlock className='text-xs-center'>
+      <p>This feature is available starting from {getXoaPlan(available)} Edition</p>
+      <a href={`https://xen-orchestra.com/#!/pricing?pk_campaign=xoa_${getXoaPlan()}_upgrade&pk_kwd=${place}`} className='btn btn-primary btn-lg'>
```

Comment:
Is it mandatory? I mean, those a 2 different things already (one text and one button)